lua/nvim-tree/view.lua

```diff
@@ -246,14 +246,20 @@ end
 local function grow()
   local starts_at = M.is_root_folder_visible(require("nvim-tree.core").get_cwd()) and 1 or 0
   local lines = vim.api.nvim_buf_get_lines(M.get_bufnr(), starts_at, -1, false)
-  local max_length = M.View.initial_width
+  local padding = 3
```

Comment:
No magic numbers please. Let's consider `view.signcolumn`.

Reply:
Out of interest: why 3 instead of 2?

Reply:
https://github.com/nvim-tree/nvim-tree.lua/blob/3ce0a8e770f70a135ef969e0a640bd8d05baf42c/lua/nvim-tree/view.lua#L251

I just used the preexisting 3 :).

Just to clarify, the intended behavior is to always have a right-padding of 1 character regardless of `view.signcolumn` and/or adaptive_size, is that right?

I have exams in the next couple of days, so I might not be very active, but I will try to fix these ASAP.